app/views/pre_registrations/edit.html.erb

```diff
@@ -1,132 +1,156 @@
+            <div class="col-md-6">
+                <%= f.label :weekend_id, 'WeekEnd Scheme' do %>
+                    <%= 'Weekend Scheme' %>
+                    <span class='required-asterisk'>*</span>
+                <% end %>
+                <p>
+                    <%= f.select :weekend_id, options_for_select(@weekend_scheme.collect { | weekend | [weekend.name,  weekend.id] }, selected: @pre_registration.weekend_id), {}, {required: 'true', autofocus: 'true', class: 'form-control', id:'select-other-projects'}%>
+                </p>
+            </div>
         </div>
 
-        <div class="col-md-6">
-            <p>
-                <%= label :weekend_id, 'WeekEnd' %>
-                <%= label '', ' *', class: 'required-asterisk' %>
-            </p>
-            <p>
-                <%= f.select :weekend_id, options_for_select(@weekend_scheme.collect { | weekend | [weekend.name,  weekend.id] }, selected: @pre_registration.weekend_id), {}, {required: 'true', autofocus: 'true', class: 'form-control', id:'select-other-projects'}%>
-            </p>
-        </div>
+        <div class="row">
 
-        <div class="col-md-6">
-            <p>
-                <%= label :NdaSigned, 'NDA Signed' %>
-            </p>
-            <p>
-                <%= f.check_box :NdaSigned, :checked => @pre_registration.NdaSigned %>
-            </p>
-        </div>
+            <div class="col-md-6">
+                <%= f.label :NdaSigned, 'NDA Signed' do %>
+                    <%= 'NDA Signed' %>
+                    <span class='required-asterisk'>*</span>
+                    <%= f.check_box :NdaSigned, :checked => @pre_registration.NdaSigned, :required => true %>
+                    <% if !@pre_registration.ndaDoc.present? %>    
+                        <%= f.file_field :ndaDoc %>
+                    <% end %>
+                <% end %>
+            </div>
 
-        <div class="col-md-6">
-            <p>
-                <%= label :workstationReady, 'Workstation Ready' %>
-            </p>
-            <p>
-                <%= f.check_box :workstationReady, :checked => @pre_registration.workstationReady %>
-            </p>
-        </div>
+            <div class="col-md-6">
+                <%= f.label :workstationReady, 'Workstation prepared for new employee?' do %>
+                    <%= 'Workstation prepared for new employee? ' %>
+                    <span class='required-asterisk'>*</span>
+                    <%= f.check_box :workstationReady, :checked => @pre_registration.workstationReady, :required => true %>
+                <% end %>
+            </div>
+
+            <div class="col-md-6">
+                <%= f.label :packReady, 'Greeting pack prepared for new employee?' do %>
+                    <%= 'Greeting pack prepared for new employee?' %>
+                    <span class='required-asterisk'>*</span>
+                    <%= f.check_box :packReady, :checked => @pre_registration.packReady, :required => true %>
```

Comment:
Change the hash syntax

Reply:
changed apu
please check it once again just to make sure whether I've used proper syntax or not